src/app-layer-htp.c

```diff
@@ -511,6 +511,7 @@ struct {
     { "C-E gzip has abnormal value", HTTP_DECODER_EVENT_ABNORMAL_CE_HEADER},
     { "C-E deflate has abnormal value", HTTP_DECODER_EVENT_ABNORMAL_CE_HEADER},
     { "C-E unknown setting", HTTP_DECODER_EVENT_ABNORMAL_CE_HEADER},
+    { "Ambiguous C-L value", HTP_REQUEST_INVALID_C_L},
```

Comment:
this table is meant to map between the string warning and the suricata internal decoder event id. So this should map to a new HTTP_DECODER_EVENT_

Then the other new part would be the rule for that warning in rules/http-events.rules + a suricata-verify test for the mix.

Reply:
Ok, see #3698, I decided to create a new event instead of reusing `HTTP_DECODER_EVENT_INVALID_CONTENT_LENGTH_FIELD_IN_REQUEST`

Reply:
I'm not sure if its relevant but I think this is too long for an event ID name. Aren't we following some char limit on each line?